tests/build/build.go

```diff
@@ -123,6 +128,241 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 
 		})
 
+		When("a new simple build component is created", Label("annotations"), func() {
+			var component *appservice.Component
+
+			BeforeAll(func() {
+				componentObj := appservice.ComponentSpec{
+					ComponentName: annotationsComponentName,
+					Application:   applicationName,
+					Source: appservice.ComponentSource{
+						ComponentSourceUnion: appservice.ComponentSourceUnion{
+							GitSource: &appservice.GitSource{
+								URL:      helloWorldComponentGitSourceURL,
+								Revision: "",
+							},
+						},
+					},
+				}
+				component, err = f.AsKubeAdmin.HasController.CreateComponent(componentObj, testNamespace, "", "", applicationName, false, nil)
+				Expect(component).ToNot(BeNil())
+				GinkgoWriter.Printf("component name: %s'\n", component.Name)
+				Expect(err).ShouldNot(HaveOccurred())
+			})
+
+			AfterAll(func() {
+				err = f.AsKubeAdmin.CommonController.Github.DeleteRef(helloWorldComponentGitSourceRepoName, annotationsPurgeBranchName)
+				if err != nil {
+					Expect(err.Error()).To(ContainSubstring("Reference does not exist"))
+				}
+			})
+
+			It("initial build triggers a pipelinerun", func() {
+				timeout = time.Minute * 1
+				interval = time.Second * 1
```

Comment:
Since the these two variables is being used in other `It` blocks, I think it would be good to define these two variables `timeout` and `interval` under `When` block